Exiled.API/Features/Pickups/Projectiles/FlashbangProjectile.cs

```diff
@@ -60,7 +60,7 @@ public float AdditionalBlindedEffect
         }
 
         /// <summary>
-        /// Gets or sets the how mush the FlashBang going to be intensified when explode at <see cref="RoomType.Surface"/>.
+        /// Gets or sets the how much the FlashBang going to be intensified when exploding on <see cref="RoomType.Surface"/>.
```

Comment:
```suggestion
        /// Gets or sets the how much the flashbang going to be intensified when exploding on <see cref="RoomType.Surface"/>.
```
Match casing from previous file changes

Reply:
Fixed